keps/0008-operator-testing.md

```diff
@@ -151,7 +151,9 @@ type TestSuite struct {
 	ManifestsDirs     []string `json:"manifestsDirs"`
 	// Directories containing test cases to run.
 	TestDirs          []string
-	// Whether or not to start a local etcd and kubernetes API server for the tests (cannot be set with StartKIND
+	// Kubectl specifies a list of kubectl commands to run prior to running the tests.
+	Kubectl []string `json:"kubectl"`
```

Comment:
can we name this `KubectlCmd` then or something like that?

Reply:
I'd originally had `kubectlCommands` but thought it was verbose/confusing (as a user: "should it be pluralized? camel-cased? did they spell out "commands" or not?"), so shortened to just `kubectl`. What do you think?

Reply:
IMO let's discuss this one more on the implementation and we can adjust it there.